### PR TITLE
julia: Use installer.script instead of original installer

### PR DESCRIPTION
**Pull request by niheaven**
`julia-installer.exe` is a NSIS installer.

### CODE REVIEW

File: bucket/julia.json [hunk 1]
```diff
@@ -13,22 +13,11 @@
...
-        "file": "uninstall.exe",
-        "args": "/S"
-    },
-    "env_set": {
-        "JULIA_HOME": "$dir\\bin",
-        "JULIA_EXE": "julia.exe",
-        "JULIA": "$dir\\bin\\julia.exe",
-        "JL_PRIVATE_LIB_DIR": "bin"
-    },
     "bin": [
```
Comment (Ash258) [inline bucket/julia.json]: Irrelevant rename
```suggestion
    "bin": "bin\\julia.exe",
```

File: bucket/julia.json [hunk 2]
```diff
@@ -13,22 +13,11 @@
...
+        "script": [
+            "Expand-7zipArchive \"$dir\\julia-installer.exe\" \"$dir\" -Removal",
+            "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall.exe\" -Recurse -Force"
         ]
     },
-    "uninstaller": {
-        "file": "uninstall.exe",
-        "args": "/S"
-    },
-    "env_set": {
```
Comment (Ash258) [inline bucket/julia.json]: Why to remove these sets?

Comment (niheaven) [reply bucket/julia.json]: I've installed julia using original installer, and it didn't set these environment variables. These may be set by previous version, and didn't be used by current one.

**Comment by niheaven** [comment]
Done.

**Pull request merged**